addie/autoNOM/run_step1.py

```diff
@@ -82,7 +82,7 @@ def retrieve_last_incremented_index(self, list_folder):
             _folder_split = _folder.split('_')
             if len(_folder_split) > 1:
                 try:
-                    _int_value = int(_folder_split[1])
+                    _ = int(_folder_split[1])
```

Comment:
Please add a comment that the code is checking that the value can be made into an integer

Reply:
8eec7e0f955b9d38312900f8ebe58f27a1aaef18